### DISCUSSION TITLE
Route(.modal/.fullScreen) background

### DISCUSSION THREAD

**Discussion by kerollesroshdi** [rundfunk47/stinsen#101] [Q&A]
Hi, 
many thanks for this great library.

I just need an example of presenting a modal/sheet view with a clear background, or can I change the presented view's default corner radius?
related question: can I remove the default background of a fullScreen Route? to be like a popup over the current view.

Thanks in advance.

**Comment by LePips** [comment]
For clear background, this is a duplicate of https://github.com/rundfunk47/stinsen/issues/60.

For changing the modal corner radius, this is not possible as that is handled by the system.

**Reply by kerollesroshdi** [reply]
so, there is no way to have a modal with a clear background?
